### PR TITLE
Control characters in bin2unicode

### PR DESCRIPTION
**Pull request by zachthompson**
For a single control character request in binary, tell user what it is (can't see it).  Otherwise, don't return answer.

Hopefully fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2783

IA Page: https://duck.co/ia/view/bin2unicode

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@zachthompson Can you add a test for the query in https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2783?: `000000000000000000000000000000000000000000000000`

File: lib/DDG/Goodie/Bin2Unicode.pm [hunk 1]
```diff
@@ -14,19 +14,99 @@ triggers query => qr{^([01\s]{8,})(?:\s+(?:to\s+)?(?:unicode|text|ascii))?$};
...
+my $zaahirs_hideout = '0' x 48;
+
 handle matches => sub {
     my $q = $_; # orginal query
     my $bin_string = shift @_; # captured binary string
 
+    my $str;
+    if($bin_string eq $zaahirs_hideout){
+        $str = q{Congratulations, you've discovered Zaahir's hideout!};
+        goto DONE;
```
Comment (moollaza) [inline lib/DDG/Goodie/Bin2Unicode.pm]: lmao thanks @zachthompson, this is great. Also, I had no idea Perl has a `goto`

Comment (zachthompson) [inline lib/DDG/Goodie/Bin2Unicode.pm]: Oh yes :trophy:

**Comment by zachthompson** [comment]
@mintsoft that query is aok, but we test for '00000000' and '00000000 00000000' now.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
> FWIW the beta deploy doesn't appear to have actually worked, but I tested locally

**Pull request merged**